dotnet/src/Connectors/Connectors.HuggingFace/HuggingFacePromptExecutionSettings.cs

```diff
@@ -344,4 +376,6 @@ public override PromptExecutionSettings Clone()
     private bool _useCache = true;
     private bool _waitForModel = false;
     private bool? _details;
+    private bool? _returnFullText;
+    private bool? _doSample;
```

Comment:
Neither of the new fields has `true` set as a default value, even though their properties claim True as the default.

Reply:
We don't need to send the default values to the server for optional parameters. We're doing this in the OpenAI connector also and the plan is to  undo that.